about.php

```diff
@@ -1,7 +1,12 @@
+<?php
+require_once "php/git.php";
+?>
+
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 	<title>OpenFlights: About</title>
+	<meta name="version" content="<?php echo(Git::getCurrentCommitID() ?? ""); ?>">
```

Comment:
My only outstanding question here is whether having a version of an empty string would look odd, but in practice, I imagine it's unlikely to happen...

Reply:
`unavailable`?